airflow/plugins_manager.py

```diff
@@ -145,11 +145,12 @@ def load_entrypoint_plugins():
     for entry_point in entry_points:  # pylint: disable=too-many-nested-blocks
         log.debug('Importing entry_point plugin %s', entry_point.name)
         try:
-            plugin_obj = entry_point.load()
-            if is_valid_plugin(plugin_obj):
+            plugin_class = entry_point.load()
+            if is_valid_plugin(plugin_class):
+                plugin_obj = plugin_class()
                 if callable(getattr(plugin_obj, 'on_load', None)):
                     plugin_obj.on_load()
-                    plugins.append(plugin_obj)
+                    plugins.append(plugin_obj())
```

Comment:
I'm not sure, but do we call here an instantiated class (via `__call__`)? Why we do this?

Reply:
We call the class constructor.Previously, we only used the class, not the plugin object.
```
>>> class AirflowTestPropertyPlugin(AirflowPlugin):
...     name = "test_property_plugin"
...     @property
...     def operators(self):
...         from airflow.models.baseoperator import BaseOperator
...         class PluginPropertyOperator(BaseOperator):
...             pass
...         return [PluginPropertyOperator]
...
>>> print(AirflowTestPropertyPlugin().operators)
[<class '__main__.AirflowTestPropertyPlugin.operators.<locals>.PluginPropertyOperator'>]
>>> print(AirflowTestPropertyPlugin.operators)
<property object at 0x10b2172c8>

```

Reply:
Currently we have:
```python
plugin_class = entry_point.load() # class object
plugin_obj = plugin_class() # class instance created via calling __init__
plugin_obj() # we call existing instance so it has to have __call__
```
What am I missing?

Reply:
Good point. Fixed.